src/screens/Exchange/Exchange.js

```diff
@@ -691,12 +730,34 @@ class ExchangeScreen extends React.Component<Props, State> {
             </CardInnerRow>
           </CardRow>
           <CardRow>
+            {!!isFiat &&
+            <CardColumn style={{ flex: 1 }}>
+              <CardText label>Fees Total</CardText>
+              <View style={{ flexDirection: 'row' }}>
+                <CardText>{feeAmount !== '' ? `${feeAmount + extraFeeAmount} ${fromAssetCode}` : 'Check'}</CardText>
```

Comment:
if no fees then what "Check" in fees means?

Reply:
Couldn't get the sendwyre api to work and give a quote, just the rates between the pairs. Check on Payment there is a screen in the widget of the price breakdown. Will have to speak with UI how to handle it better.

Reply:
So maybe then if it's unable to get fees then don't show this part?

Reply:
I'd let it stay just because it will imply fees and the user should have the heads up for that